examples/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -46,6 +47,11 @@ const {
 
 const { TITLE_TEXT, OUTPUT_DEVICE_BUTTON } = soundOutputDevicesLabels;
 
+const { OUTPUT_DEVICES_BOTTOM_SHEET } = soundOutputDevicesLabels;
+
+// TODO: Remove after fixing: https://linear.app/swmansion/issue/FCE-504/fix-e2e-test-on-ios-related-to-showing-camera-view-when-camera-is
```

Comment:
nit: you can just leave ticket number:
```suggestion
// TODO: Remove after fixing: FCE-504
```